.github/workflows/observability-charm-tests.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
         run: |
           sed -i -e "/^ops[ ><=]/d" -e "/canonical\/operator/d" -e "/#egg=ops/d" requirements.txt
           echo -e "\ngit+$GITHUB_SERVER_URL/$GITHUB_REPOSITORY@$GITHUB_SHA#egg=ops" >> requirements.txt
+          echo -e "\nmarkupsafe==2.0.1" >> requirements.txt  # temp fix for https://github.com/canonical/grafana-k8s-operator/pull/69
```

Comment:
We can remove this later when https://github.com/canonical/grafana-k8s-operator/pull/69 is merged, but it makes our CI fail so is needed to merge.

Reply:
This is merged now, reverting this hack.